en/theme/material/assets/css/redoc.css

```
@@ -0,0 +1,198 @@
+}
+
+.md-content__inner {
+    margin: 0;
+    padding-top: 0;
+}
+
+.md-main__inner {
+    margin-top: 0;
+}
+
+[dir=ltr] .md-typeset ol li,
```

Stylelint (17.2.0)

[error] Expected quotes around "ltr" (selector-attribute-quotes)

(selector-attribute-quotes)